websvc.saas.codegen.java/src/org/netbeans/modules/websvc/saas/codegen/java/resources/ApiKeyAuthenticator.template

```diff
@@ -1,3 +1,24 @@
+<#--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
```

Comment:
This .template file should probably be excluded the nbbuild/build.xml "rat" task since it's user visible.